src/ape/api/compiler.py

```diff
@@ -51,6 +51,45 @@ def compile(
             List[:class:`~ape.type.contract.ContractType`]
         """
 
+    @raises_not_implemented
+    def get_imports(
+        self, contract_filepaths: List[Path], base_path: Optional[Path]
+    ) -> Dict[str, List[str]]:
+        """
+        Returns a list of imports for each contract in a given compiler.
+
+        Args:
+            contract_filepaths (List[pathlib.Path]): A list of source file paths to compile.
+            base_path (Optional[pathlib.Path]): Optionally provide the base path, such as the
+              project ``contracts/`` directory. Defaults to ``None``. When using in a project
+              via ``ape compile``, gets set to the project's ``contracts/`` directory.
+
+        Returns:
+            Dict[str, List[str]]
+        """
+
+    def _get_filename_dict_from_paths(
```

Comment:
Is this the right location for this method?
It seems to be a static, internal method that does not depend on any `CompilerManager` state nor is it used anywhere in this module.

Reply:
Method removed